test/RBTreeTest.mo

```diff
@@ -536,6 +575,19 @@ run(
           tree
         },
         TreeMatcher([])
+      ),
+      test(
+        "unshare",
+        do {
+          let o = buildTestTree();
+          let saved = o.share();
+          o.unshare(#leaf);
+          let empty = o.share();
+          o.unshare(saved);
+          let restored = o.share();
+          empty == #leaf and restored == saved;
+        },
+        M.equals(T.bool(true))
```

Comment:
Why is this same unit test (`unshare`) repeated three separate times?  Three different `buildTestTree` implementations, it seems?

It would be less confusing for future reviews if they each had a different suffix, at least (`Tiny`, `Small`, `Big`).

Also would help if matchers would permit more code reuse between these otherwise-identical unit tests.

Reply:
Yeah, I didn't write the original test but the buildTestTree is a var that gets updated with different build functions.

Reply:
Life is too short.

Reply:
I'll name apart in a separate PR. Wanted toget this out for a new release